staker-rewards/README.adoc

```diff
@@ -1,35 +1,112 @@
 = Staker rewards
 
-== Prerequisites
+ECDSA staker rewards are based on ETH locked and KEEP staked over time. The
+rewards calculations are performed weekly off-chain, and committed for
+distribution using a variant of Uniswap’s Merkle distributor. This repository
+contains scripts for calculating rewards and generation of Merkle tree which
+can be used by stakers to claim their rewards for a given interval. Anyone
+can use these scripts to validate reward distribution or to generate reward
+estimates for a pending interval.
 
-To run the script, make sure you have the following prerequisites:
+== Algorithm
 
-- Node.js (at least v14.3.0). We recommend to use
-  the https://github.com/nvm-sh/nvm[Node Version Manager] to get the right
-  Node.js version with no troubles. You can install this tool by running
-  `brew install nvm` on macOS or follow the https://github.com/nvm-sh/nvm#installing-and-updating[installation guide]
-  for other operating systems.
+The algorithm for calculating ECDSA staker rewards as implemented in the scripts 
+in this repository is described in this section.
 
-== How to run
+=== Input
+* Interval `T` starts at `T_start`.
 
-1. Make sure you use the right Node.js version:
-+
-```
-nvm use 14.3.0
-```
-2. Install the NPM dependencies if not done yet:
-+
+* Interval `T` ends at `T_end`.
+
+* All operators who have ever bonded ETH are taken into account.
```

Comment:
bonded -> deposited ?
```suggestion
* All operators who have ever deposited ETH are taken into account.
```

Reply:
What about `deposited into the bonding contract`? Don't want to confuse stakers who think they e.g. deposit to operator's account.

Reply:
Sounds perfect!